File: DivisibleByFiveOrNot.java
import java.util.*;

class Mathematic
{
    public boolean Dividation(int No)
    {
        if(No % 5 == 0)
        {
            return true;
        }
        else
        {
            return false;
        }
    }
}

class DivisibleByFiveOrNot
{
    public static void main(String arg[])
    {
        Scanner sobj = new Scanner(System.in);

        int iNo = 0;
        boolean bRet = false;

        System.out.println("Enter the number:");
        iNo = sobj.nextInt();

        Mathematic mobj = new Mathematic();
        
        bRet =mobj.Dividation(iNo);
        if(bRet == true)
        {
            System.out.println(iNo + " number is divided by 5.");
        }
        else 
        {
            System.out.println(iNo + " number is not divided by 5.");
        }

        
    }
}

File: DivisibleByFiveOrNot.cpp
#include<iostream>
using namespace std;

class DivisibleByFiveOrNot
{
    public:
      int No;

      DivisibleByFiveOrNot(int iValue)
      {
          No = iValue;
      }

      bool Dividation()
      {
          if(No%5 ==0)
          {
              return true;
          }
          else
          {
              return false;
          }
      }

};

int main()
{
    int iNo = 0;
    bool bRet = false;

    cout<<"Enter the number:\n";
    cin>>iNo;

    DivisibleByFiveOrNot obj(iNo);

    bRet=obj.Dividation();

    cout<<iNo<<" number is divisible by 5."<<"\n";

    return 0;

}